crossbar/worker/router.py

```diff
@@ -607,7 +606,8 @@ def _create_resource(reactor, path_config, templates, log, cbdir, _router_sessio
                 raise ApplicationError(u"crossbar.error.invalid_configuration", emsg)
             else:
                 try:
-                    static_dir = os.path.abspath(pkg_resources.resource_filename(path_config['package'], path_config['resource']))
+                    static_dir = os.path.abspath(os.path.join(os.path.dirname(
+                        importlib.import_module(path_config['package']).__file__), path_config['resource']))
                 except Exception as e:
```

Comment:
Can we use `pkgutil.get_data()` here, as described here http://pyinstaller.readthedocs.io/en/latest/spec-files.html#using-data-files-from-a-module

Reply:
I looked deeper into this and found that pkutil.get_data() is only used to get data files and **not** directories, any such attempt throws a IsADirectoryError exception. But the good thing is pkg_resources.resource_filename() is working fine for pyinstaller provided `datas`, so I reverted those changes.

Reply:
Ah, okay, good to know.